CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(QuickTurns VERSION 0.1 LANGUAGES CXX)
+
+set(CMAKE_AUTOUIC ON)
```

Comment:
Do we need it?

Reply:
This command enables automatic generation of C++ code from Qt Designer UI files. Since we are not going to use the Qt Designer, probably we can consider removing `set(CMAKE_AUTOUIC ON)`.